src/components/layout/Navbar/style.module.scss.d.ts

```diff
@@ -1,4 +1,6 @@
 export type Styles = {
+  mobileNavItem: string | undefined;
```

Comment:
is this the auto-generated code? I don't think the class names can be undefined since they're coming from the scss file. (I couldn't figure out a good way to generate these types on save in the codebase so it generates them automatically whenever you restart and run `yarn dev`. stretch goal to fix that 👀 )